### PR TITLE
setup dotenv

### PR DESCRIPTION
**Pull request by Kalimbak**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    test: {
+        port: process.env.TEST_PORT,
+        DB: process.env.TESTDB,
+        JWT_SECRET: process.env.JWT_SECRET
+    },
+    production: {
+        port: process.env.PORT,
+        DB: process.env.DEVDB,
+        JWT_SECRET: process.env.JWT_SECRET
+    },
```
Comment (furebo) [inline config.js]: For DB in production you need to make DATABASE_URL as environment variable. Because it will be used in production instead of DEVDB.

**Review by furebo — COMMENTED** [review]
Also inside the src folder create a folder called config then move config.js file inside it.

**Review by kwizeradev — APPROVED** [review]

**Review by Angelus123 — APPROVED** [review]

**Review by tjeaneric — APPROVED** [review]

**Review by furebo — COMMENTED** [review]
Please reduce the number of commits to one commit

**Review by furebo — APPROVED** [review]
Please reduce the number of commits to one commit

**Pull request merged**